test/unit/spec/v2/view-builder/utils/LinksUtil_spec.js

```diff
@@ -0,0 +1,43 @@
+import AppState from 'v2/models/AppState';
+import { FORMS } from 'v2/ion/RemediationConstants';
+import { getSwitchAuthenticatorLink } from 'v2/view-builder/utils/LinksUtil';
+
+describe('v2/utils/LinksUtil', function() {
+  const mockAppState = (remediationFormName, hasMoreThanOneAuthenticator) => {
+    const appState = new AppState();
+    spyOn(appState, 'hasRemediationObject').and.callFake(formName => formName === remediationFormName);
+    spyOn(appState, 'hasMoreThanOneAuthenticatorOption').and.returnValue(hasMoreThanOneAuthenticator);
```

Comment:
Let's make sure we use `jest` spies instead of `jasmine`:

```suggestion
    jest.spyOn(appState, 'hasRemediationObject').mockImplementation(formName => formName === remediationFormName);
    jest.spyOn(appState, 'hasMoreThanOneAuthenticatorOption').mockReturnValue(hasMoreThanOneAuthenticator);
```

Reply:
Bye jasmine